anonymizer/base.py

```diff
@@ -303,6 +303,9 @@ def alter_object_attribute(self, obj, attname, replacer):
     def run(self, chunksize=2000, parallel=4):
         self.validate()
 
+        if not self.create_replacer_attributes():
```

Comment:
This is a speed improvement (short circuit).
This is a bug fix because the update query is malformed when it updates 0 columns.